README.md

```diff
@@ -264,19 +264,19 @@ Similarly to dups and sups, lambdas and applications can have labels too.
 For example, data types can be encoded as tagged lambdas:
 ```rs
 // data Bool = T | F
-T = λ#Bool t λ#Bool f t
-F = λ#Bool t λ#Bool f f
+T = #Bool @t #Bool @f t
+F = #Bool @t #Bool @f f
 
 // data List = (Cons x xs) | Nil
-Cons = λx λxs λ#List c λ#List n (#List.Cons.xs (#List.Cons.x c x) xs)
-Nil  =        λ#List c λ#List n n
+Cons = @x @xs #List @c #List @n #List (c x xs)
```

Comment:
Here the application should be `#List.Cons.xs (#List.Cons.x (c x) xs)`

Reply:
also, ins'nt a little better to use `λ` to mantain consistence on the examples?

Reply:
It is correct, this is showing how to encode data types with tagged lambdas, not what a `data` definition is desugared into

Reply:
Fixed, I had not noticed this correctly before ;)